tooling/autodispose-rx-error-prone-checker/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+  id "net.ltgt.errorprone" version "0.0.13"
+  id "com.github.johnrengelman.shadow" version "2.0.1"
```

Comment:
can you explain the shadow bits here?

Reply:
I am trying to create a fat jar here and the steps are taken from - [NullAway's build.gradle - dependency part](https://github.com/uber/NullAway/blob/master/nullaway/build.gradle). 

It is not working in any case as I am not able to use it in the sample project. Will be fixing it over the weekend. I am not sure what is the best way to publish an error-prone check.

Reply:
Hmm. Maybe @msridhar or @kageiit have advice here

Reply:
You should not need to create a fat jar. the jar with just the classes from this module can be consumed by anyone that uses error prone

Reply:
Yeah all the shadow stuff isn't needed here. NullAway needs it due to a forked dependence

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/05a3a34b3778da0031e5dfa0161256f224ed6cdf